chips/earlgrey/src/aes.rs

```diff
@@ -275,6 +294,10 @@ impl<'a> Aes<'a> {
 
 impl<'a> hil::symmetric_encryption::AES128<'a> for Aes<'a> {
     fn enable(&self) {
+        #[cfg(feature = "config_sim_verilator")]
+        {
+            self.wait_on_idle().unwrap();
```

Comment:
No `unwrap()`, at the very least an expect.

Reply:
I removed, this entirely (turns out we don't need the wait for enable)